pagopa/io-app
# chore: [IOAPPFD0-18,IOAPPFD0-23] Fix e2e tests

## Short description
This PR try to fix the e2e tests running on the CI.

## List of changes proposed in this pull request
- package.json: bumped `detox` from `19.7.0` to `19.13.0`. This version fixes a bug while using `Xcode 14.x`.
- yarn.lock: bumped `detox` from `19.7.0` to `19.13.0`. This version fixes a bug while using `Xcode 14.x`.
- ts/__e2e__/payment.e2e.ts:  fixed wrong `testId` label
- ts/__e2e__/config.ts: increased `e2eWaitRenderTimeout` from `10` to `15`  to avoid false positive tests due to missing views for timing issues.

## How to test
Run e2e locally and see that all works fine.

If you wanna try running test on this PR you must comment the `master` filter [here](https://github.com/pagopa/io-app/blob/7938ce2ffe15c9ec294f37650ff867fc16ddf9b4/.circleci/config.yml#L535).

| [This test has been already done on this PR then reverted](https://app.circleci.com/pipelines/github/pagopa/io-app/24835/workflows/4720e219-7595-4038-9dc5-4c6e076d2794/jobs/138556) | 
| - | 
| ![Screen Shot 2023-01-23 at 17 10 23](https://user-images.githubusercontent.com/16268789/214090461-29d001d0-5084-4ccf-97f0-01747447b159.png) |
